### DISCUSSION TITLE
What is the purpose of *_sun_min_elevation in settings.yml file?

### DISCUSSION THREAD

**Discussion by airlomba** [jekhokie/raspberry-noaa-v2#483] [Q&A] [answered]
Hello all,

I have been tweaking my raspi-noaa-v2 lately on a fresh installation, and since I am not interested in the NOAA photos at night, I was expecting that the sun minimum elevation setting would work as for the satellite minimum elevation; i.e., only receive the pass if the sun is above horizon (day).

Am I missing something here? Thank you for any input.

Best regards from Portugal.
Emmanuel
(CT7AFR).

**Comment by dom-robinson** [comment, marked as answer]
It is used to determine which NOAA enhancements are performed on the processing of the capture..

**Reply by airlomba** [reply]
Oh! That makes perfect sense now.
Thank you for your explanation.
Regards,
Emmanuel.
